implementations/rust/ockam/ockam_transport_ble/src/driver/bluetooth_hci/mod.rs

```diff
@@ -92,7 +92,7 @@ where
         }
     }
 
-    pub fn reset<T, Time>(&mut self, timer: &mut T, time: Time) -> ockam::Result<()>
+    pub fn reset<T, Time>(&mut self, timer: &mut T, time: Time) -> ockam_core::Result<()>
```

Comment:
Is there a reason why we can't make `use ockam_core::Result`?

Reply:
Ah yes, at one point there were a lot more `Result` types scattered through the code before I broke the various bluetooth bits into their own drivers and cleaned things up. 

No point in naming them explicitly anymore now though so I've gone and cleaned this up as well now.